bin/sqs_cli.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# SQS manipulation script.
+#
+# Note: Run this in the base container which has access to SQS.
+#
+# Usage: ./bin/sqs_cli.py [SUBCOMMAND]
```

Comment:
This script is pretty general and because of that, we should be able to just copy it into Socorro.